### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by fky2015**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/------.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**预期的行为是什么**
+A clear and concise description of what you expected to happen.
+
+**截图**
+If applicable, add screenshots to help explain your problem.
+
+**系统环境 (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (phy25) [inline .github/ISSUE_TEMPLATE/------.md:28]: Is this needed?

**Review by phy25 — CHANGES_REQUESTED** [review]
I know this is a draft but...

Any chance to rename this file (might be generated through the Chinese title)?

**Comment by fky2015** [comment]
> I know this is a draft but...
> 
> Any chance to rename this file (might be generated through the Chinese title)?

Will do.

**Comment by everything411** [comment]
LGTM

**Review by everything411 — APPROVED** [review]
LGTM

**Pull request merged**